.env.example

```
@@ -70,6 +70,10 @@ NEXT_PUBLIC_ZENDESK_KEY=
 # Help Scout Config
 NEXT_PUBLIC_HELPSCOUT_KEY=
 
+# Inbox to send user feedback
+SEND_FEEDBACK_EMAIL=support@cal.com
```

Let's not leave a default email in here, we don't want to receive support from self-hosted instances that don't bother to change this.

```suggestion
SEND_FEEDBACK_EMAIL=
```

---

fair enough